charts/sn-platform/templates/broker/broker-cluster.yaml

```diff
@@ -281,6 +285,12 @@ spec:
 {{ toYaml . | indent 6 }}
 {{- end }}
       {{- if and (.Values.auth.vault.enabled) (.Values.auth.oauth.enabled) }}
+      {{- if and .Values.components.functions .Values.broker.functionmesh.enabled }}
+      # enable package service for function mesh
+      packagesManagementStorageProvider: "{{ .Values.broker.packagesManagement.storageProvider }}"
+      packagesReplicas: "{{ .Values.broker.configData.managedLedgerDefaultEnsembleSize }}"
```

Comment:
Maybe we only need to configure this for now, and we can configure it in the operator.